hack/allure-report.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+
+CE=${CE:-podman}
+
+$CE info 2>/dev/null >/dev/null || NO_CE=$?
+if [ -n "$NO_CE" ]; then
+    echo "WARNING $CE is required but not installed, skipping..."
+    exit 0
```

Comment:
This can make the issue silent and not visible. How about that we exit here with `exit 1`?

Reply:
good pint